code/modules/mob/living/simple_animal/animals/synx.dm

```diff
@@ -32,7 +32,7 @@
 				"bomb" = 10,
 				"bio" = 100,
 				"rad" = 100)
-	has_hands = TRUE
+	has_hands = 1

```

Comment:
One and the same, aren't they?

Reply:
Evrywhere else 0/1 is used not True False, its more a fix to be aligned with how its supposed to look.